### ISSUE TITLE
cupy-core can be installed with cupy<13

### ISSUE THREAD

**Issue by carterbox** [conda-forge/cupy-feedstock#256]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

If you install `cupy=13` then roll back to `cupy<13`, the `cupy-core` package remains in the environment. Probably the `cupy-core` package needs a run_constraint on `cupy >=13` so that it is not installable with `cupy` packages that predate it.

### Installed packages

```shell
# packages in environment at /home/beams/DCHING/miniconda3/envs/tike-tomo:
#
# Name                    Version                   Build  Channel
cupy                      12.3.0          py311h134e05b_2    conda-forge
cupy-core                 13.0.0          py311heecd119_3    conda-forge
```

### Environment info

```shell
active environment : tike-tomo
    active env location : /home/beams/DCHING/miniconda3/envs/tike-tomo
            shell level : 2
       user config file : /home/beams/DCHING/.condarc
 populated config files : /home/beams/DCHING/miniconda3/.condarc
                          /home/beams/DCHING/.condarc
          conda version : 23.11.0
    conda-build version : 3.28.2
         python version : 3.10.13.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=broadwell
                          __conda=23.11.0=0
                          __cuda=11.2=0
                          __glibc=2.28=0
                          __linux=4.18.0=0
                          __unix=0=0
       base environment : /home/beams/DCHING/miniconda3  (writable)
      conda av data dir : /home/beams/DCHING/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/beams/DCHING/miniconda3/pkgs
                          /home/beams/DCHING/.conda/pkgs
       envs directories : /home/beams/DCHING/miniconda3/envs
                          /home/beams/DCHING/.conda/envs
               platform : linux-64
             user-agent : conda/23.11.0 requests/2.31.0 CPython/3.10.13 Linux/4.18.0-240.el8.x86_64 rhel/8.4 glibc/2.28 solver/libmamba conda-libmamba-solver/23.12.0 libmambapy/1.5.6
                UID:GID : 1992:3234
             netrc file : /home/beams/DCHING/.netrc
           offline mode : False
```

**Comment by carterbox** [comment]
That seems right. I always forget that we can patch the metadata of already published packages. I've never done a repodata patch before.

**Comment by leofang** [comment]
I haven't done that since the patches were converted to the new format, I'll figure it out later...

**Comment by leofang** [comment]
Fixed in dd16c72.
